SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -11,10 +11,14 @@
 Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
 > To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
 
+## Glossary
```

Comment:
"This workflow is very subject to change" - we can remove that I think

Reply:
Removed.